experiment-setup/bootstrap/test/server/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM docker:27.3.1-dind-alpine3.20
```

Comment:
You are using dind here, young padawan. The dark side of the force. A true jedi searches his own heart and uses Docker sockets instead. Don't try to control the force. Simply use it.

Reply:
Hahahaha I knew you would say something. I will take a look on your PR to get some inspiration :)

Reply:
How did you do that? I didn't understand. Can you give me some tips how to remove dind and use this sockets you mentioned? @Ghmeding